packages/clay-management-toolbar/src/ClayManagementToolbar.soy

```diff
@@ -0,0 +1,240 @@
+	{@param? handlePlusButtonClicked_: any}
+	{@param? handleSelectAllClicked_: any}
+	{@param? handleSortingButtonClicked_: any}
+	{@param? id: string}
+	{@param? selectable: bool}
+	{@param? selectedItems: number}
+	{@param? sortingOrder: string}
+	{@param? totalItems: number}
+	{@param? viewTypes: list<?>}
+
+	{let $isActive: $selectedItems and $selectedItems > 0 /}
+
+	{if $isActive}
+		{call .active}
+			{param actionItems: $actionItems /}
+			{param elementClasses: $elementClasses /}
+			{param handleSelectPageCheckboxChanged_: $handleSelectPageCheckboxChanged_ /}
+			{param handleSelectAllClicked_: $handleSelectAllClicked_ /}
+			{param selectedItems: $selectedItems /}
+			{param spritemap: $spritemap /}
+			{param totalItems: $totalItems /}
+		{/call}
+	{else}
+		{call .default}
+			{param elementClasses: $elementClasses /}
+			{param filterItems: $filterItems /}
+			{param handleFilterDoneButtonClick_: $handleFilterDoneButtonClick_ /}
+			{param handleSelectPageCheckboxChanged_: $handleSelectPageCheckboxChanged_ /}
+			{param handlePlusButtonClicked_: $handlePlusButtonClicked_ /}
+			{param handleSortingButtonClicked_: $handleSortingButtonClicked_ /}
+			{param id: $id /}
+			{param selectable: $selectable /}
+			{param sortingOrder: $sortingOrder /}
+			{param spritemap: $spritemap /}
+			{param viewTypes: $viewTypes /}
+		{/call}
+	{/if}
+{/template}
+
+/**
+ * This renders the component's active content.
+ */
+{template .active}
+	{@param spritemap: string}
+	{@param? actionItems: list<?>}
+	{@param? elementClasses: string}
+	{@param? handleSelectPageCheckboxChanged_: any}
+	{@param? handleSelectAllClicked_: any}
+	{@param? id: string}
+	{@param? selectedItems: number}
+	{@param? totalItems: number}
+
+	{let $navAttributes kind="attributes"}
+		class="management-bar management-bar-light navbar navbar-expand-md
+			{if $elementClasses}
+				{sp}{$elementClasses}
+			{/if}
+		"
+
+		{if $id}
+			id="{$id}"
+		{/if}
+	{/let}
+
+	<nav {$navAttributes}>
+		<div class="container">
+			<ul class="navbar-nav">
+				<li class="nav-item">
+					{call ClayCheckbox.render}
+						{param checked: true /}
+						{param events: ['change': $handleSelectPageCheckboxChanged_] /}
+						{param hideLabel: true /}
+						{param indeterminate: $selectedItems < $totalItems /}
+						{param label: 'Select Page' /}
+						{param ref: 'selectPageCheckbox' /}
+					{/call}
+				</li>
+				<li class="nav-item">
+					<span class="navbar-text">
+						{$selectedItems} of {$totalItems}{sp}
+						<span class="navbar-breakpoint-down-d-none">items selected</span>
+					</span>
+				</li>
+				<li class="nav-item">
+					{call ClayButton.render}
+						{param elementClasses: 'nav-btn' /}
+						{param events: ['click': $handleSelectAllClicked_] /}
+						{param label: 'Select all' /}
+						{param ref: 'selectAllButton' /}
+						{param style: 'link' /}
+					{/call}
+				</li>
+			</ul>
+			
+			{if $actionItems}
+				<ul class="navbar-nav">
```

Comment:
Hey @carloslancha, is not it necessary to add `Quick actions`? defined by the lexicon team?

<img width="135" alt="screen shot 2017-11-28 at 11 35 09" src="https://user-images.githubusercontent.com/13750819/33325135-3b6e3be6-d430-11e7-8666-e65816197b58.png">

Reply:
I did not see the above comment, sorry.